fornax-hea/build-ciao.sh

```diff
@@ -50,6 +50,36 @@ ln -sf $SUPPORT_DATA_DIR/ciao-${CIAO_VERSION}/spectral/modelData $ENV_DIR/ciao/s
 # link caldb.
 ln -sf $SUPPORT_DATA_DIR/ciao-caldb-${CIAO_VERSION}/CALDB $ENV_DIR/ciao/CALDB
 
+
+# Writing scripts to ensure the CALDB and HEASoft PFILES paths are properly set up when the CIAO environment is loaded
+####### CALDB #######
+# Start with the CALDB activation and deactivation scripts - necessary because we're not using the CIAO CALDB conda
+#  package, we already have CALDB living somewhere else, and as such the conda activate.d script that calls the
+#  CALDB init script does not exist
+cat << EOF > $ENV_DIR/ciao/etc/conda/activate.d/caldb_main_activate.sh
+export CALDB=$CONDA_PREFIX/CALDB
```

Comment:
In case CONDA_PREFIX is not defined yet when this is called, use $ENV_DIR/ciao/CALDB